eng/pipelines/performance/templates/perf-arm64-build-jobs.yml

```diff
@@ -12,7 +14,7 @@ jobs:
         buildConfig: release
         runtimeFlavor: mono
         platforms:
-        - linux_arm64
+        - ${{ if eq(parameters.linux, true) }}: linux_arm64
```

Comment:
```suggestion
        - ${{ if eq(parameters.linux, true) }}:
          - linux_arm64
```
Convention is typically to put the list entries on a separate line, I believe this is the correct syntax for it